### ISSUE TITLE
Nicely formatted function-offsets for addresses

### ISSUE THREAD

**Issue by Shaddy** [tmr232/Sark#21]
I'm not sure if this already exists, but I didn't find it. The following is not probably the best implementation but represents the goal.

```
def get_symbolic_name(ea):
    """Returns function offset string.
    """
    function = sark.Function(ea)
    if function.ea == ea:
        return function.name
    else:
        return "{0.name}+{1:X}".format(function, ea - function.ea)

In [203]: for xref in sark.Function(name='SomeFunction').xrefs_to:
     ...:     print get_symbolic_name(xref.frm)
     ...:     
sub_4060B7+17
SomeFunctionName+16
```

**Comment by tmr232** [comment]
What would you have it return outside a function?
Throw an exception?
Return an offset from the nearest name?

**Comment by Shaddy** [comment]
Sorry for the delay, I think that the same that IDA does when you check "Function offsets" as an address representation. `{section}:{offset}`

**Comment by tmr232** [comment]
Started implementing this, and got into annoying design issues... This feature needs a `Line` object to get both a `Segment` and a `Function`. This is problematic since both of those import the `Line` module.
I am not yet sure how I want to handle this.

**Comment by tmr232** [comment]
I added a `Line.offset_name` property in another branch - https://github.com/tmr232/Sark/tree/offset-name

The code is still a bit of a mess, but is this what you want?

**Comment by Shaddy** [comment]
Seems that finally you handled the design issue!

Thats exactly what I want :+1: :) :)

**Comment by tmr232** [comment]
Merged in  f4ce6d2
